client/gutenberg/extensions/presets/jetpack/utils/jetpack-logo.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { Path, Polygon, SVG } from '@wordpress/components';
+
+const JetpackLogo = () => (
+	<SVG
+		width={ 24 }
+		height={ 24 }
+		viewBox="0 0 32 32"
+	>
+		<Path
+			fill="#00be28"
+			d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z"
+		/>
+		<Polygon fill="#fff" points="15,19 7,19 15,3 " />
+		<Polygon fill="#fff" points="17,29 17,13 25,13 " />
```

Comment:
There's a tweaked version for 24px size:

https://github.com/Automattic/wp-calypso/blob/241292989efa898345417b4632f6402605497895/client/components/jetpack-logo/index.jsx#L38-L45

If that is truly the size that Gutenberg uses, we should consider swapping it. If I recally correctly, @shaunandrews may have details on whether/why this tweaked version was/continues to be necessary 🙇